docs/operators/from_http.md

```diff
@@ -13,8 +13,9 @@ from_http url:string, [method=string, payload=string, headers=record,
           max_retry_count=int, retry_delay=duration, responses=record,
           max_request_size=int, tls=bool, certfile=string, keyfile=string,
           password=string { … }]
-from_http url:string, server=true, [responses=record, max_request_size=int,
-          tls=bool, certfile=string, keyfile=string, password=string { … }]
+from_http url:string, server=true, [metadata_field=field, responses=record,
+          max_request_size=int, tls=bool, certfile=string, keyfile=string,
+          password=string { … }]
```

Comment:
Can you double-check the two signatures? The client one has `responses` which I don't think it should, but please take a look all other options for both modes as well.

Reply:
Removed `responses` and `max_request_size`. I think the others are fine.